src/olympia/scanners/admin.py

```diff
@@ -430,8 +436,21 @@ def get_urls(self):
         return custom_urls + urls
 
     def result_actions(self, obj):
+        info = self.model._meta.app_label, self.model._meta.model_name
         return render_to_string(
-            'admin/scannerresult_actions.html', {'obj': obj}
+            'admin/scannerresult_actions.html', {
+                'handlefalsepositive_urlname': (
+                    'admin:%s_%s_handlefalsepositive' % info
+                ),
+                'handletruepositive_urlname': (
+                    'admin:%s_%s_handletruepositive' % info
+                ),
+                'handleinconclusive_urlname': (
+                    'admin:%s_%s_handleinconclusive' % info
+                ),
+                'handlerevert_urlname': 'admin:%s_%s_handlerevert' % info,
+                'obj': obj,
+            }
```

Comment:
Do we want to triage the code search results as well?

Reply:
I'm not sure yet. I wanted the actions to work on a basic level, like not impact regular scanner results at the very least. Whether we end up actually needing the actions is still up in the air